x/evm/precompile/wasm_test.go

```diff
@@ -34,11 +38,48 @@ func TestWasmSuite(t *testing.T) {
 	suite.Run(t, new(WasmSuite))
 }
 
-func (s *WasmSuite) TestExecuteHappy() {
+func (s *WasmSuite) TestInstantiate() {
```

Comment:
I added this because I tore out the instantiating via wasm precompile in the `SetupWasmContracts()` test method.

In general, it's bad practice to rely on test setup code to validate behavior of production code.